README.md

```diff
@@ -80,6 +80,7 @@ Other dedicated linters that are built-in are:
 | [luacheck][19]      | `luacheck`     |
 | [chktex][20]        | `chktex`       |
 | [vint][21]          | `vint`         |
+| nix                 | `nix`          |
```

Comment:
Can you link this to the [Github page](https://github.com/Synthetica9/nix-linter) ?

Reply:
This does not use `nix-linter`, it uses `nix-instantiate --parse` which is built-in to Nix.

Reply:
I didn't check, sorry about that